### PR TITLE
Adding namespace flag to kedge build

### LINKED ISSUE #574: `kedge build` doesn't allow namespace/project to be specified
`kedge build` doesn't allow namespace/project to be specified.  This causes confusion when multiple kedge operations are running parallelly especially during the testing.

```
$ ./kedge build -h
build application image

Usage:
  kedge build [flags]

Flags:
  -b, --builder-image string   Name of a Docker image to use as a builder. Note: This is only useful when using s2i build strategy
  -c, --context string         Path to a directory containing a Dockerfile, it is build context that is sent to the Docker daemon (default ".")
  -f, --file string            Specify Dockerfile for doing builds, Dockerfile path is relative to context (default "Dockerfile")
  -i, --image string           Image name and tag of resulting image
  -p, --push                   Add this flag if you want to push the image. Note: Ignored when s2i build strategy used
      --s2i      

```

### PR DESCRIPTION
**Pull request by surajnarwade**
This PR will allow `kedge build` to specify flag for namespace.
Resolves #574

### CODE REVIEW

**Comment by cdrage** [comment]
Code LGTM. Just got to test it.

**Comment by ashetty1** [comment]
@kadel could you merge this? I would like to send a PR updating OpenShift tests on top of this.

**Comment by surajnarwade** [comment]
@kadel review needed :)

File: pkg/build/build.go [hunk 1]
```diff
@@ -320,7 +323,7 @@ func BuildS2I(image, context, builderImage string) error {
 	}
 
 	log.Infof("Starting build for %q", image)
-	cmd := []string{"oc", "start-build", image, "--from-dir=" + context, "-F"}
+	cmd := []string{"oc", "start-build", image, "--from-dir=" + context, "-F", "--namespace", namespace}
```
Comment (kadel) [inline pkg/build/build.go]: I'm not sure that this will work if a namespace is not set.

Comment (ashetty1) [reply pkg/build/build.go]: @kadel it works:

```
# ./kedge build --s2i -i s2i -b centos/python-35-centos7
imagestream "s2i" created
buildconfig "s2i" created
INFO[0012] Starting build for "s2i"
build "s2i-1" started

Pushing image 172.30.1.1:5000/myproject/s2i:latest ...
Pushed 0/10 layers, 0% complete
Pushed 1/10 layers, 10% complete
Pushed 2/10 layers, 21% complete
Pushed 3/10 layers, 32% complete
Pushed 4/10 layers, 42% complete
Pushed 5/10 layers, 55% complete
Pushed 6/10 layers, 65% complete
Pushed 7/10 layers, 87% complete
Pushed 8/10 layers, 90% complete
Pushed 9/10 layers, 95% complete
Pushed 10/10 layers, 100% complete
Push successful


# oc get all
NAME      TYPE      FROM      LATEST
bc/s2i    Source              1

NAME           TYPE      FROM             STATUS     STARTED         DURATION
builds/s2i-1   Source    Binary@e47a415   Complete   5 minutes ago   4m24s

NAME      DOCKER REPO                     TAGS      UPDATED
is/s2i    172.30.1.1:5000/myproject/s2i   latest    43 seconds ago

NAME             READY     STATUS      RESTARTS   AGE
po/s2i-1-build   0/1       Completed   0          5m
```

**Comment by ashetty1** [comment]
ping @surajnarwade, can we get this merged, please. I have a few changes to push on top of this :)

**Comment by surajnarwade** [comment]
@kadel can we merge this

**Pull request merged**